.github/ISSUE_TEMPLATE/comp_exporter_geneva.md

```diff
@@ -0,0 +1,42 @@
+---
+name: OpenTelemetry.Exporter.Geneva
+about: Issue with OpenTelemetry.Exporter.Geneva
+labels: comp:exporter.geneva
+---
+
+# Issue with OpenTelemetry.Exporter.Geneva
+
+List of [all OpenTelemetry NuGet
+packages](https://www.nuget.org/profiles/OpenTelemetry) and version that you are
+using (e.g. `OpenTelemetry 1.0.2`):
+
+* [OpenTelemetry.Exporter.Geneva
+  1.2.6](https://www.nuget.org/packages/OpenTelemetry.Exporter.Geneva/1.2.6)
+
+Runtime version (e.g. `net462`, `net48`, `netcoreapp3.1`, `net6.0` etc. You can
+find this information from the `*.csproj` file):
+
+* TBD
+
+**Is this a feature request or a bug?**
```

Comment:
I'd happy to steal a lot from https://github.com/dotnet/aspnetcore/blob/main/.github/ISSUE_TEMPLATE/10_bug_report.yml

Reply:
I think the issue with high structured template is that folks will just ignore/bypass it, e.g. https://github.com/dotnet/aspnetcore/issues/41385 and https://github.com/dotnet/aspnetcore/issues/41383

Reply:
> We will close this issue if:
> 
> The repro project you share with us is complex. We can't investigate custom projects, so don't point us to such, please.
> If we will not be able to repro the behavior you're reporting.
> If the repro project is attached as a .zip file.
> If the GitHub repro project is set to Private.

^^ This alone is worth copying. Agree that a very long template would be tempting to ignore..

Reply:
These seem to be too aggressive IMHO 🤣:

> If the repro project is attached as a .zip file.
> If the GitHub repro project is set to Private.